### PR TITLE
Add O-RAN Integration documentation

### PR DESCRIPTION
**Pull request by adetalhouet**

### CODE REVIEW

File: content/en/docs/network-architecture/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: Network Architecture
+description: Nephio Release Scope, User Stories and Roadmap
+weight: 2
+---
+
+{{% pageinfo %}}
+This section is to formalize the work delivered as part of SIG1 Network Architecture.
+{{% /pageinfo %}}
+
+SIG1 Network Architecture focuses on defining and developing network architecture identifying and/or exercicing Nephio's enablers. The group's work is guided by user stories that capture the needs and priorities of stakeholders, such as network administrators, security officers, engineers, and end-users.
```
Comment (liamfallon) [inline content/en/docs/network-architecture/_index.md:11]: ``
```suggestion
SIG1 Network Architecture focuses on defining and developing network architecture as well as identifying and/or exercicing Nephio's enablers. The group's work is guided by user stories that capture the needs and priorities of stakeholders, such as network administrators, security officers, engineers, and end-users.
```

File: content/en/docs/network-architecture/_index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+The scope includes establishing guidelines for network design, security protocols, scalability solutions, performance optimization, and providing comprehensive documentation and training. 
+
+This website contains only a subset, approved contant, of the produced materials. Everything is available for anyone's consumption in [Nephio SIG1 Google Drive](https://drive.google.com/drive/u/2/folders/1P4bmDppUIrtBbiRe3fj92xA_qlCMn7sf).
```
Comment (liamfallon) [inline content/en/docs/network-architecture/_index.md:14]:
```suggestion
This website contains approved content, which is a subset of the material produced by SIG1. Everything is available for anyone's consumption in [Nephio SIG1 Google Drive](https://drive.google.com/drive/u/2/folders/1P4bmDppUIrtBbiRe3fj92xA_qlCMn7sf).
```

File: content/en/docs/network-architecture/_index.md (hunk 1 above)
Comment (adetalhouet) [reply content/en/docs/network-architecture/_index.md:11]: The point here is to say the network architecture is identifying and/or exercising the Nephio's enablers. The later part of the sentence shouldn't be read as another thing SIG1 is doing. This is done as part of the network architecture.

File: content/en/docs/network-architecture/user-stories/Create O-Cloud Node Cluster.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+title: Create O-Cloud Node Cluster
+description: >
+  User Story about O-RAN O2 IMS Provisioning
+weight: 5
+---
+
+ <div class="container">
+  <iframe class="responsive-iframe" src="https://docs.google.com/presentation/d/e/2PACX-1vTCgOufcrbqNHkopzOVS_yuvcN2H4ZDJc-LEGEiSAhSIUWS0_2h4IZ5GP0Eq0WXqg/embed?start=false&loop=false&delayms=3000" frameborder="0" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
```
Comment (CsatariGergely) [inline content/en/docs/network-architecture/user-stories/Create O-Cloud Node Cluster.md:9]: We should avoid adding Google docs or wiki pages to the documentation. These do not support proper versioning. 

For me the content of the Google slides looks like a design document. I'm not sure if we should publish design documents in the documentation pages.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: content/en/docs/network-architecture/_index.md (hunk 1 above)
Comment (liamfallon) [reply content/en/docs/network-architecture/_index.md:11]: Would this read better?

SIG1 Network Architecture focuses on defining and developing network architecture ***by*** identifying and/or exercising Nephio's enablers.

Comment (adetalhouet) [reply content/en/docs/network-architecture/_index.md]: Fixed

File: content/en/docs/network-architecture/user-stories/Create O-Cloud Node Cluster.md (hunk 3 above)
Comment (adetalhouet) [reply content/en/docs/network-architecture/user-stories/Create O-Cloud Node Cluster.md]: Agreed. We ended up providing documentation for the O-RAN Integration. No longer referencing any gdoc.

**Review by liamfallon — COMMENTED** [review]
/lgtm
/approve

File: content/en/docs/network-architecture/O-RAN Integration.md
Comment (tcareyintx) [inline content/en/docs/network-architecture/O-RAN Integration.md:1]: Is O-RAN Integration.md a correct convention for naming documentation files? seems the convention is lowercase with dashes. So maybe oran-integration.md?

File: content/en/docs/network-architecture/O-RAN Integration.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+# Using Nephio enablers to realize O-RAN functionality
+## Overview
+The O-RAN Software Community (OSC) and Open Air Interface (OAI) and Nephio open source communities are working together to provide a reference implementation of the Open RAN (O-RAN) Alliance’s cloud-centric specifications using the Nephio enablers and capabilities in order to deploy and manage O-RAN NFs and xApps.
+The focus of the O-RAN Integration within Nephio focuses on the Federated O-Cloud Orchestration and Management (FOCOM), Network Function Orchestration (NFO), Infrastructure Management (IMS) and Deployment Management (DMS) O-RAN services as depicted in the following figure:
+
+![deployment-architecture.png](/static/images/network-architecture/o-ran/deployment-architecture.png)
+
+These services can be categorized into:
+-Infrastructure services for the lifecycle management of Cloud Resources (FOCOM, IMS): These services are deployed and utilizes the Nephio capabilities provided by Nephio Management Clusters. In some deployed architectures aspects of the IMS utilizes Nephio capabilities provided by Nephio Workload Clusters.
```
Comment (tcareyintx) [inline content/en/docs/network-architecture/O-RAN Integration.md:34]: This did not get a bullet missing space on the dash.

File: content/en/docs/network-architecture/_index.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ weight: 2
 This section is to formalize the work delivered as part of SIG1 Network Architecture.
 {{% /pageinfo %}}
 
-SIG1 Network Architecture focuses on defining and developing network architecture identifying and/or exercicing Nephio's enablers. The group's work is guided by user stories that capture the needs and priorities of stakeholders, such as network administrators, security officers, engineers, and end-users.
+SIG1 Network Architecture focuses on defining and developing network architecture by identifying and/or exercicing Nephio's enablers. The group's work is guided by user stories that capture the needs and priorities of stakeholders, such as network administrators, security officers, engineers, and end-users.
```
Comment (tcareyintx) [inline content/en/docs/network-architecture/_index.md:11]: spelling excercicing?

File: content/en/docs/network-architecture/O-RAN Integration.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+## Infrastructure services for the lifecycle management of Cloud Resources
+### Introduction
+The primary role of the FOCOM and IMS services is to provide for the lifecycle management of the resources exposed by an O-Cloud. These services have identified the following O-Cloud capabilities that are planned to be developed using Nephio capabilities:
+- O-Cloud registration
+- O-Cloud Cluster Lifecycle Management
+- O-Cloud Resource Inventory
+
+> **NOTE:**  In R3 Nephio use cases and component architectures have been defined for O-Cloud Cluster Lifecycle Management
+As stated, the role of the FOCOM function is to provide federated orchestration and management across multiple O-Clouds using the O2ims interface between the O-Clouds as shown in the figure below:
```
Comment (tcareyintx) [inline content/en/docs/network-architecture/O-RAN Integration.md:45]: This statement got mixed in with the previous Note. Should be it's own statement.

File: content/en/docs/network-architecture/O-RAN Integration.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+In O-RAN, a cloudified O-RAN NF (e.g., O-DU) is deployed and managed in an O-Cloud using one or more NF Deployments. While the NFO’s focus is the lifecycle management of the deployed O-RAN NF, the DMS’s focus is the NF Deployment which is also the capabilities provided by Nephio. 
+NF Deployment Lifecycle Management services implemented using Nephio capabilities are described in the following use cases:
+- Deploy O-RAN NF
+- Terminate O-RAN NF Deployments
+
+> **NOTE:**  In R3 Nephio use cases and component architectures have been defined for the Deploy O-RAN NF use case.
+
+##### Deploy O-RAN NF
+The role of the NFO within the SMO is to handle functionality related to the lifecycle management for deployment of O-RAN Cloudified NFs. The deployment of the O-RAN NF includes software components realizing all or part of an NF which could be an O-DU, O-CU-CP, O-CU-UP, xApps and the Near-RT RIC components in the O-Cloud sites.
+This use case describes the high-level steps and requirements for deployment of O-RAN NF that is composed as discrete NF Deployments across the O2dms using the K8s profile.
```
Comment (tcareyintx) [inline content/en/docs/network-architecture/O-RAN Integration.md:117]: This should be a separate paragraph. Markup keeps it with the previous sentence.

**Comment by liamfallon** [comment]
I see @tcareyintx made some comments after the PR was merged. @adetalhouet if changes are needed based on @tcareyintx comments, I guess we could raise another PR with those changes.

File: content/en/docs/network-architecture/O-RAN Integration.md
Comment (adetalhouet) [reply content/en/docs/network-architecture/O-RAN Integration.md:1]: fixed here https://github.com/nephio-project/docs/pull/147

File: content/en/docs/network-architecture/O-RAN Integration.md (hunk 4 above)
Comment (adetalhouet) [reply content/en/docs/network-architecture/O-RAN Integration.md:34]: fixed here https://github.com/nephio-project/docs/pull/147

File: content/en/docs/network-architecture/_index.md (hunk 5 above)
Comment (adetalhouet) [reply content/en/docs/network-architecture/_index.md:11]: fixed here https://github.com/nephio-project/docs/pull/147

File: content/en/docs/network-architecture/O-RAN Integration.md (hunk 6 above)
Comment (adetalhouet) [reply content/en/docs/network-architecture/O-RAN Integration.md:45]: fixed here https://github.com/nephio-project/docs/pull/147

File: content/en/docs/network-architecture/O-RAN Integration.md (hunk 7 above)
Comment (adetalhouet) [reply content/en/docs/network-architecture/O-RAN Integration.md:117]: fixed here https://github.com/nephio-project/docs/pull/147

**Comment by adetalhouet** [comment]
@tcareyintx Thanks for the review. Unfortunately the PR was merged to soon. I fixed the outstanding comments here fixed here https://github.com/nephio-project/docs/pull/147

**Pull request merged**